### PR TITLE
Warning Fix: Updated use of parseHtml function

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR fixes the warning in parseHtml function.
Reference: https://stackoverflow.com/a/37905107

Also, to check this PR correctly, run `TopicListControllerTest` and run the app and check that the HomeActivity is displayed correctly.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

File: domain/src/main/java/org/oppia/domain/topic/TopicListController.kt [hunk 1]
```diff
@@ -349,11 +349,7 @@ class TopicListController @Inject constructor(
   }
 
   private fun parseHtml(html: String): Spannable {
-    return if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
-      Html.fromHtml(html, Html.FROM_HTML_MODE_LEGACY) as Spannable
-    } else {
-      Html.fromHtml(html) as Spannable
-    }
+    return HtmlCompat.fromHtml(html, HtmlCompat.FROM_HTML_MODE_LEGACY) as Spannable
```
Comment (veena14cs) [inline domain/src/main/java/org/oppia/domain/topic/TopicListController.kt:352]: Please update this in HtmlParser class too under utility package.

**Review by veena14cs — COMMENTED** [review]
Update all the files that uses Html.fromHtml() function.

**Review by veena14cs — APPROVED** [review]
LGTM

**Pull request merged**